eth_utils/conversions.py

```diff
@@ -19,8 +23,11 @@
 )
 
 
+Primitives = Union[bytes, int, bool]
+
+
 @validate_conversion_arguments
-def to_hex(primitive=None, hexstr=None, text=None):
+def to_hex(primitive: Primitives=None, hexstr: str=None, text: str=None) -> str:
```

Comment:
It *might* be good for us to define a `HexStr = NewType('HexStr', str)` so that we can differentiate hex strings from normal strings.